### PR TITLE
51-fix-icons-color

### PR DESCRIPTION
**Pull request by Duelsik**
Screenshot from 2021-12-01 21-35-40

### CODE REVIEW

File: src/components/HourlyForecast/HourlyForecast.tsx [hunk 1]
```diff
@@ -64,9 +64,10 @@ const HourlyItem = (forecast: HourWeatherForecast) => {
 
 type HourlyForecastProps = Pick<TodayWeatherProps, "hourForecast">;
 const HourlyForecast = ({ hourForecast }: HourlyForecastProps) => {
-  const forecast = React.useMemo(() => hourForecast.filter(skipFirst()).filter(head(8)).flatMap(HourlyItem), [
-    hourForecast
-  ]);
+  const forecast = React.useMemo(
+    () => hourForecast.filter(skipFirst()).filter(head(8)).flatMap(HourlyItem),
+    [hourForecast]
+  );
```
Comment (Duelsik) [inline src/components/HourlyForecast/HourlyForecast.tsx:70]: (prettier)

File: src/index.tsx [hunk 2]
```diff
@@ -6,8 +6,8 @@ import "./index.css";
 // This enables HMR
 // TODO: check if it affects prod build
 // sorry TS
-if (((module as unknown) as Record<string, unknown>)?.hot) {
-  ((module as unknown) as Record<"hot", { accept: () => void }>)?.hot?.accept();
+if ((module as unknown as Record<string, unknown>)?.hot) {
+  (module as unknown as Record<"hot", { accept: () => void }>)?.hot?.accept();
```
Comment (Duelsik) [inline src/index.tsx:10]: (prettier)

File: src/assets/icons/fa-arrow.svg [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
-  <path fill="#FFF" d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"/>
+  <path fill="#000" d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"/>
```
Comment (Duelsik) [inline src/assets/icons/fa-arrow.svg:2]: Calculating filters assumes that icon is black.

Comment (Cutler7) [reply src/assets/icons/fa-arrow.svg:2]: You probably are right

File: src/common/cssProp.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import type { Properties } from "csstype";
+
+import { css } from "styled-components";
+import { prop } from "./helpers";
+
+export const cssProp =
+  <T, K extends keyof T>(cssName: keyof Properties, attrName: K, defaultValue = "") =>
+  (props: T) => {
+    console.log(cssName, attrName, defaultValue, prop<T, K>(attrName, defaultValue)(props));
```
Comment (Cutler7) [inline src/common/cssProp.ts:9]: Is this console log stayed here on purpose?

File: src/common/hexToCssFilter.ts [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  if (result) {
+    return [parseInt(result[1], 16), parseInt(result[2], 16), parseInt(result[3], 16), parseInt(result[4], 16)];
+  }
+
+  throw new Error("Error parsing hex: " + hex);
+}
+
+export function hexToCssFilter(hex: string) {
```
Comment (Cutler7) [inline src/common/hexToCssFilter.ts:332]: very complex but isolated - ok

File: src/index.tsx [hunk 6]
```diff
@@ -6,8 +6,8 @@ import "./index.css";
 // This enables HMR
 // TODO: check if it affects prod build
 // sorry TS
-if (((module as unknown) as Record<string, unknown>)?.hot) {
-  ((module as unknown) as Record<"hot", { accept: () => void }>)?.hot?.accept();
+if ((module as unknown as Record<string, unknown>)?.hot) {
```
Comment (Cutler7) [inline src/index.tsx:9]: TS was crying during compilation :P If these unknowns necessary - ok, but some comment here would be nice.

**Review by Cutler7 — APPROVED** [review]

**Pull request merged**